frontend/src/pages/dashboard/ManageData.jsx

```diff
@@ -0,0 +1,361 @@
+import React, { useState, useEffect, useCallback } from "react";
+import AdvanceFilter from "../../components/filter";
+import {
+  Row,
+  Col,
+  Table,
+  Select,
+  Tabs,
+  Spin,
+  Button,
+  notification,
+} from "antd";
+import { UIState } from "../../state/ui";
+import { generateAdvanceFilterURL } from "../../util/utils";
+import { api } from "../../lib";
+import { DownloadOutlined } from "@ant-design/icons";
+import Exports from "./Exports";
+
+const { TabPane } = Tabs;
+
+const ManageData = () => {
+  const { provinceValues, advanceSearchValue, schoolTypeValues } =
+    UIState.useState((s) => s);
+  const [selectedProvince, setSelectedProvince] = useState([]);
+  const [selectedSchoolType, setSelectedSchoolType] = useState([]);
+  const [data, setData] = useState([]);
+  const [monitoringData, setMonitoringData] = useState([]);
+  const [monitoringRound, setMonitoringRound] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [tabLoading, setTabLoading] = useState(false);
+  const [exportLoading, setExportLoading] = useState(false);
+  const [paginate, setPaginate] = useState({
+    total: 1,
+    current: 1,
+    pageSize: 10,
+  });
+  const [page, setPage] = useState("database");
+
+  const [tabItems, setTabItems] = useState([]);
+
+  const handleProvinceFilter = (value) => {
+    setSelectedProvince(value);
+  };
+
+  const handleSchoolTypeFilter = (value) => {
+    setSelectedSchoolType(value);
+  };
+
+  const handleMonitoringFilter = (value) => {
+    setMonitoringRound(value);
+  };
+
+  const handleTabClick = (key) => {
+    setPage(key);
+  };
+
+  useEffect(() => {
+    const url = `option/monitoring_round`;
+
+    api
+      .get(url)
+      .then((res) => {
+        setMonitoringData(res.data);
+      })
+      .catch(() => {
+        setMonitoringData([]);
+      });
+  }, []);
+
+  const getdata = useCallback(
+    (page = 1, pageSize = 10) => {
+      setLoading(true);
+      let url = `data?page=${page}&perpage=${pageSize}`;
+      url = generateAdvanceFilterURL(advanceSearchValue, url);
+      if (selectedProvince && selectedProvince.length > 0) {
```

Comment:
Hi @navins94 , 
What do you think if we create a util to generate filter URL? I think we do the same "if" condition on more than one page.

Reply:
Sure I will create util for this

Reply:
I will create a separate branch for this. You can review the pr again